src/modules/AuthModule/components/ChangePass/ChangePass.tsx

```diff
@@ -1,8 +1,211 @@
+import Visibility from "@mui/icons-material/Visibility";
+import VisibilityOff from "@mui/icons-material/VisibilityOff";
+import { IconButton, TextField, ThemeProvider, createTheme } from "@mui/material";
+import Box from "@mui/material/Box";
+import Button from "@mui/material/Button";
+import Typography from "@mui/material/Typography";
+import { useFormik } from "formik";
+import * as React from "react";
+import { toast } from "react-toastify";
+import { axiosInstanceWithHeaders } from "../../../../axiosConfig/axiosInstance";
+import { FormChangePass } from "../../../../interfaces/Auth";
+import { useNavigate } from "react-router-dom";
+import { validationSchema } from "../../../../validations/validations";
+
+
+const defaultTheme = createTheme();
 
 const ChangePass = () => {
+  const [showOldPassword, setShowOldPassword] = React.useState(false);
+  const [showNewPassword, setShowNewPassword] = React.useState(false);
+  const [showConfirmPassword, setShowConfirmPassword] = React.useState(false);
+  const navigate = useNavigate()
+
+  const handleClickShowOldPassword = () => setShowOldPassword((show) => !show);
+  const handleMouseDownOldPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const handleClickShowNewPassword = () => setShowNewPassword((show) => !show);
+  const handleMouseDownNewPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const handleClickShowConfirmPassword = () =>
+    setShowConfirmPassword((show) => !show);
+  const handleMouseDownConfirmPassword = (
+    event: React.MouseEvent<HTMLButtonElement>
+  ) => {
+    event.preventDefault();
+  };
+
+  const style = {
```

Comment:
We can give the style the type SxProps to get autocompletion for valid sx values.
This type is exposed from the MUI library we can get it as follows:
```tsx
import type { SxProps } from '@mui/material';
```